scripts/functions/studies/studies.py

```
@@ -36,4 +36,7 @@ def delete_indexed_equipments(networkUuid):
 
 def invalidate_nodes_builds(study_uuid):
     return requests.delete(constant.DELETE_STUDY_NODES_BUILDS.format(studyUuid = study_uuid))
+
+def unmount_study(study_uuid):
+    return requests.delete(constant.UNMOUNT_STUDY.format(studyUuid = study_uuid))
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)